### PR TITLE
About Page Mobile Responsiveness

### PR DESCRIPTION
**Pull request by tresimovic**
**_WHAT DOES THIS PR DO?_**
 - Fixes some padding and font-size issues on the About Page so it displays nicely on mobile screens.
 - Before: (iPhone 12 Pro - 390x844 dimensions)
Screenshot from 2023-12-01 20-54-03

 - After: (iPhone 12 Pro - 390x844 dimensions)
Screenshot from 2023-12-01 20-55-58

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
 - https://github.com/chrisj117/CS-472-GROUP-2-PROJECT/issues/126

**_HOW DO I TEST OUT THIS PR?_**
 - Run the project locally
 - `cd web` then `npm run dev`
 - Go to http://localhost:5173/ in your web browser
 - Navigate to About Page and then inspect the page
 - Toggle the device toolbar and switch to iPhone 12 Pro dimensions
 - Updated About Page should appear

### CODE REVIEW

**Review by azboss2021 — APPROVED** [review]
Looks much better @tresimovic  thank you

**Review by chrisj117 — CHANGES_REQUESTED** [review]
In my testing, rotating the mobile screen in dark mode causes the bottom of the screen to become white as pictured:
image
However, I found that adding `overflow-auto` to the highest level `div` fixes this issue. Otherwise, this is a big improvement on mobile, and this small bug fix should be all this PR needs.

File: web/src/pages/About.jsx [hunk 1]
```diff
@@ -1,31 +1,33 @@
-import ReturnHome from '../components/ReturnHome';
+import ReturnHome from "../components/ReturnHome";
 
 const About = () => {
   return (
-    <div className="flex flex-col items-center h-[calc(100vh-94px)] w-screen max-w-screen-xl mx-auto">
-      <h2 className="text-3xl font-bold mb-12 mt-8">About</h2>
+    <div className="flex flex-col items-center h-[calc(100vh-94px)] w-screen mx-auto px-4 md:max-w-screen-xl">
```
Comment (chrisj117) [inline web/src/pages/About.jsx:5]: This line is specifically where I applied `overflow-auto` to fix the dark mode mobile bug mentioned in my review.

**Comment by danielogen** [comment]
The changes look great so far...

**Review by carrec-2 — CHANGES_REQUESTED** [review]
Great work! Not sure if you caught some of these errors already or if you fixed them yet, but here are some issues I found. 
1. 
home page 1
In the about page In dark mode, all the text is there but the dark mode gets cutoff on the vertical iphone 12 pro dimensions. 
2.  
home page 2
In the about page In dark mode, only the first part of text is there and the rest gets cutoff with white in the horizontal iphone 12 pro dimensions
3.  
home page 4
home page 5
On the home page in both light/dark mode, the  "Find your school below to leave an evaluation!" is missing in the horizontal iphone 12 pro dimensions
4.  
home page 6
On the login page in dark mode, the dark mode gets cutoff in the horizontal iphone 12 pro dimensions.

**Review by chrisj117 — APPROVED** [review]
With the navbar changes from PR #135 , the horizontal mode's "white bar in dark mode" bug seems to have been fixed. After the merge conflicts are solved this PR should also be good to go 👍

**Review by carrec-2 — APPROVED** [review]

**Comment by tresimovic** [comment]
I have fixed the bug related to dark mode on a rotated mobile screen, thanks for catching it.

**Pull request merged**